### PR TITLE
docs: fix out of dated description in autoloader.rst

### PR DESCRIPTION
**Pull request by kenjis**
**Description**
- fix out-of-dated description on App namespace
- update sample code

**Checklist:**
- [x] Securely signed commits
- [] Component(s) with PHPDoc blocks, only if necessary or adds value
- [] Unit testing, with >80% coverage
- [x] User guide updated
- [] Conforms to style guide

### CODE REVIEW

**Review by datamweb — APPROVED** [review]

**Comment by kenjis** [comment]
Pushed other updates.

File: user_guide_src/source/concepts/autoloader/003.php [hunk 1]
```diff
@@ -1,5 +1,15 @@
...
-];
+namespace Config;
+
+use CodeIgniter\Config\AutoloadConfig;
+
+class Autoload extends AutoloadConfig
+{
+    // ...
+    public $classmap = [
+        'Markdown' => APPPATH . 'third_party/markdown.php',
```
Comment (datamweb) [inline user_guide_src/source/concepts/autoloader/003.php:11]:
```suggestion
        'Markdown' => APPPATH . 'ThirdParty/markdown.php',
```

Comment (kenjis) [reply user_guide_src/source/concepts/autoloader/003.php]: Thank you!

**Review by michalsn — APPROVED** [review]

File: user_guide_src/source/concepts/autoloader.rst [hunk 2]
```diff
@@ -47,25 +47,26 @@ Namespaces
...
+The value is the location to the directory the classes can be found in.
 
 .. note:: You can check the namespace configuration by ``spark namespaces`` command::
 
     > php spark namespaces
 
-By default, the application folder is namespace to the ``App`` namespace. While you are not forced to namespace the controllers,
-libraries, or models in the application directory, if you do, they will be found under the ``App`` namespace.
+By default, the application folder is namespace to the ``App`` namespace. You must namespace the controllers,
+libraries, or models in the application directory, and they will be found under the ``App`` namespace.
```
Comment (samsonasik) [inline user_guide_src/source/concepts/autoloader.rst:63]: usage of `folder` or `directory` may need to be consistent, eg: `application folder` used and next used `application directory`

Comment (kenjis) [reply user_guide_src/source/concepts/autoloader.rst:63]: Done.

**Comment by samsonasik** [comment]
Thank you @kenjis

**Pull request merged**